src/read_utils.cu

```diff
@@ -440,7 +440,14 @@ std::unique_ptr<SceneInfo> read_colmap_scene_info(std::filesystem::path file_pat
         sceneInfos->_point_cloud = read_ply_file(file_path / "sparse/0/points3D.ply");
     }
     sceneInfos->_ply_path = file_path / "sparse/0/points3D.ply";
-    sceneInfos->_cameras = read_colmap_cameras(file_path / "images", cameras, images);
+
+    auto& cams = sceneInfos->_cameras = read_colmap_cameras(file_path / "images", cameras, images);
+    const float image_mpixels = cams[0]._img_w * cams[0]._img_h / 1000000.0f;
```

Comment:
And maybe name this constant 1'000'000.f  If you use these upper ticks, than its easier to read (for instance, see parameters.cuh)